unstructured/ingest/v2/README.md

```diff
@@ -0,0 +1,119 @@
+  - [ ] Register the required information via `add_source_entry` or `add_source_entry` to expose the new connectors.
+- [ ] Update the CLI to expose the new connectors via CLI params
+  - [ ] Add a new file under [cmds](unstructured/ingest/v2/cli/cmds)
+  - [ ] Add the command base classes from the file above in the [__init__.py](unstructured/ingest/v2/cli/cmds/__init__.py). This will expose it in the CLI.
+- [ ] Update [unstructured/ingest/cli](unstructured/ingest/cli) with support for the new connector.
+- [ ] Create a folder under [examples/ingest](examples/ingest) that includes at least one well documented script.
+- [ ] Add a script test_unstructured_ingest/[src|dest\/test-ingest-\<the-new-data-source\>.sh. It's json output files should have a total of no more than 100K.
+- [ ] Git add the expected outputs under test_unstructured_ingest/expected-structured-output/\<folder-name-relevant-to-your-dataset\> so the above test passes in CI.
+- [ ] Add a line to [test_unstructured_ingest/test-ingest.sh](test_unstructured_ingest/test-ingest.sh) invoking the new test script.
+- [ ] Make sure the tests for the connector are running and not skipped by reviewing the logs in CI.
+- [ ] If additional python dependencies are needed for the new connector:
+  - [ ] Add them as an extra to [setup.py](unstructured/setup.py).
+  - [ ] Update the Makefile, adding a target for `install-ingest-<name>` and adding another `pip-compile` line to the `pip-compile` make target. See [this commit](https://github.com/Unstructured-IO/unstructured/commit/ab542ca3c6274f96b431142262d47d727f309e37) for a reference.
+  - [ ] The added dependencies should be imported at runtime when the new connector is invoked, rather than as top-level imports.
+  - [ ] Add the decorator `unstructured.utils.requires_dependencies` on top of each class instance or function that uses those connector-specific dependencies e.g. for `GitHubConnector` should look like `@requires_dependencies(dependencies=["github"], extras="github")`
+  - [ ] Run `make tidy` and `make check` to ensure linting checks pass.
+- [ ] Update ingest documentation [here](https://github.com/Unstructured-IO/docs)
+- [ ] For team members that are developing in the original repository:
+  - [ ] If there are secret variables created for the connector tests, make sure to:
+    - [ ] add the secrets into Github (contact someone with access)
+    - [ ] include the secret variables in [`ci.yml`](https://github.com/Unstructured-IO/unstructured/blob/main/.github/workflows/ci.yml) and [`ingest-test-fixtures-update-pr.yml`](https://github.com/Unstructured-IO/unstructured/blob/main/.github/workflows/ingest-test-fixtures-update-pr.yml)
+    - [ ] add a make install line in the workflow configurations to be able to provide the workflow machine with the required dependencies on the connector while testing
```

Comment:
assuming this is the github workflow configuration for ci right? maybe link here?

Reply:
hmm, if neither of us are certain what this is referring to ... that's not good. actually digging in, wonder if this pre-dated the iteration in the `install-all-ingest` [target](https://github.com/Unstructured-IO/unstructured/blob/915aa0f386bcd5f04a55e646a51b6a87a61c7058/Makefile#L108-L111) and we required manually adding `make install` commands?

Reply:
This is related to the make targets for each connector dependency install, such as this: [install-ingest-google-drive](https://github.com/Unstructured-IO/unstructured/blob/main/Makefile#L111)